app-nordigen/nordigen-node/api/account.js.map

```diff
@@ -0,0 +1 @@
ails\n     */\n    getDetails() {\n        return this.#get(\"details\");\n    }\n\n    /**\n     * Access premium account details\n     * @returns Object with premium account details\n     */\n    getPremiumDetails() {\n        return this.#getPremium(\"details\");\n    }\n    \n    /**\n     * Access account balances\n     * @returns Object with account balances\n     */\n    getBalances() {\n      return this.#get(\"balances\");\n    }\n\n    /**\n     * @param {Object} [params]\n     * @param {string} params.dateFrom\n     * @param {string} params.dateTo\n     * Access account transactions\n     * @returns Object with account transactions\n     */\n    getTransactions({dateFrom, dateTo} = {}) {\n        const dateRange = {\n            \"date_from\": dateFrom,\n            \"date_to\": dateTo\n        }\n        return this.#get(\"transactions\", dateRange);\n    }\n\n    /**\n     * @param {Object} [params]\n     * @param {string} params.dateFrom\n     * @param {string} params.dateTo\n     * Access premium account transactions\n     * @returns Object with premium account transactions\n     */\n    getPremiumTransactions({dateFrom, dateTo} = {}) {\n        const dateRange = {\n            \"date_from\": dateFrom,\n            \"date_to\": dateTo\n        }\n        return this.#getPremium(\"transactions\", dateRange);\n    }\n}\n"],"mappings":";;;;;;;AAAe,MAAMA,UAAN,CAAiB;EAE5B,CAACC,QAAD,GAAY,UAAZ;EACA,CAACC,MAAD,GAAU,IAAV;EACA,CAACC,SAAD;EAEA;AACJ;AACA;AACA;AACA;AACA;;EACIC,WAAW,CAAC;IAACF,MAAD;IAASC;EAAT,CAAD,EAAsB;IAC7B,KAAK,CAACD,MAAN,GAAeA,MAAf;IACA,KAAK,CAACC,SAAN,GAAkBA,SAAlB;EACH;EAED;AACJ;AACA;AACA;AACA;AACA;;;EACI,CAACE,GAAG,CAACC,IAAD,EAAOC,UAAU,GAAG,EAApB,EAAwB;IACxB,MAAMC,GAAG,GAAI,GAAE,KAAK,CAACP,QAAS,IAAG,KAAK,CAACE,SAAU,IAAGG,IAAK,EAAzD;IACA,OAAO,KAAK,CAACJ,MAAN,CAAaO,OAAb,CAAqB;MAACR,QAAQ,EAAEO,GAAX;MAAgBD;IAAhB,CAArB,CAAP;EACH;EAED;AACJ;AACA;AACA;AACA;AACA;;;EACI,CAACG,UAAU,CAACJ,IAAD,EAAOC,UAAU,GAAG,EAApB,EAAwB;IAC/B,MAAMC,GAAG,GAAI,GAAE,KAAK,CAACP,QAAS,YAAW,KAAK,CAACE,SAAU,IAAGG,IAAK,EAAjE;IACA,OAAO,KAAK,CAACJ,MAAN,CAAaO,OAAb,CAAqB;MAACR,QAAQ,EAAEO,GAAX;MAAgBD;IAAhB,CAArB,CAAP;EACH;EAED;AACJ;AACA;AACA;;;EACII,WAAW,GAAG;IACV,OAAO,KAAK,CAACT,MAAN,CAAaO,OAAb,CAAqB;MACxBR,QAAQ,EAAG,GAAE,KAAK,CAACA,QAAS,IAAG,KAAK,CAACE,SAAU;IADvB,CAArB,CAAP;EAGH;EAED;AACJ;AACA;AACA;;;EACIS,UAAU,GAAG;IACT,OAAO,KAAK,CAACP,GAAN,CAAU,SAAV,CAAP;EACH;EAED;AACJ;AACA;AACA;;;EACIQ,iBAAiB,GAAG;IAChB,OAAO,KAAK,CAACH,UAAN,CAAiB,SAAjB,CAAP;EACH;EAED;AACJ;AACA;AACA;;;EACII,WAAW,GAAG;IACZ,OAAO,KAAK,CAACT,GAAN,CAAU,UAAV,CAAP;EACD;EAED;AACJ;AACA;AACA;AACA;AACA;AACA;;;EACIU,eAAe,CAAC;IAACC,QAAD;IAAWC;EAAX,IAAqB,EAAtB,EAA0B;IACrC,MAAMC,SAAS,GAAG;MACd,aAAaF,QADC;MAEd,WAAWC;IAFG,CAAlB;IAIA,OAAO,KAAK,CAACZ,GAAN,CAAU,cAAV,EAA0Ba,SAA1B,CAAP;EACH;EAED;AACJ;AACA;AACA;AACA;AACA;AACA;;;EACIC,sBAAsB,CAAC;IAACH,QAAD;IAAWC;EAAX,IAAqB,EAAtB,EAA0B;IAC5C,MAAMC,SAAS,GAAG;MACd,aAAaF,QADC;MAEd,WAAWC;IAFG,CAAlB;IAIA,OAAO,KAAK,CAACP,UAAN,CAAiB,cAAjB,EAAiCQ,SAAjC,CAAP;EACH;;AArG2B"}
```

Comment:
How was this generated?

Reply:
I used Babel CLI to convert this lib - https://github.com/nordigen/nordigen-node
I tried to install just this lib and convert it on the fly to CommonJS but I lost with it... and just add it as a converted package. Maybe someone has a better experience with babel and knows how to convert lib on the fly.

Reply:
OK. I think it would make sense to port actual-server to ESM but I’d be happy to pursue that in a future PR!